### PR TITLE
Proto defs for inventory commands

### PR DESCRIPTION
**Pull request by PaulAgeyev**
Created commands for Inventory: 

- AppendInventory
- BorrowBook
- ReserveBook
- CancelReservation
- ExtendLoanPeriod
- MarkLoanOverdue
- MarkReservationExpired
- ReturnBook
- ReportLostBook
- WriteBookOf

Refactored identifiers for Inventory: 
UserId, InventoryId, InventoryItemId, LoanId.

### CODE REVIEW

File: model/src/main/proto/javaclasses/exlibris/c/commands.proto [hunk 1]
```diff
@@ -81,3 +81,147 @@ message RemoveBook {
...
+message BorrowBook {
+
+    // An identifier of an inventory item.
+    InventoryItemId intentoryItem_id = 1;
+
+    // An identifier of the user, who borrows the book.
+    UserId user_id = 2;
+}
+
+
```
Comment (alexanderkarpetc) [inline model/src/main/proto/javaclasses/exlibris/c/commands.proto:123]: Delete one

**Review by Dmytry-Dyachenko — APPROVED** [review]
LGTM

File: model/src/main/proto/javaclasses/exlibris/identifiers.proto [hunk 2]
```diff
@@ -40,17 +40,45 @@ message BookId {
...
+// An identifier is represented by based on a date.
+//
 message LoanId {
 
+    // An automatically generated identifier based on a date.
+    string value = 1;
 }
-message UserId {
-    EmailAddress email =1;
-}
```
Comment (alexanderkarpetc) [inline model/src/main/proto/javaclasses/exlibris/identifiers.proto:56]: Press enter

**Review by alexanderkarpetc — APPROVED** [review]
LGTM

File: model/src/main/proto/javaclasses/exlibris/values.proto [hunk 3]
```diff
@@ -97,5 +97,3 @@ message RFID {
 message EmailAddress {
     string value = 1;
 }
-
-
```
Comment (alexanderkarpetc) [inline model/src/main/proto/javaclasses/exlibris/values.proto:101]: remove one

**Pull request merged**